tests/Unit/Core/Form/IdentifiableObject/CommandBuilder/Product/Combination/DefaultCombinationCommandsBuilderTest.php

```diff
@@ -104,6 +104,23 @@ public function getExpectedCommands(): Generator
             [],
         ];
 
+        yield [
```

Comment:
You should also revert this test along with the modification in the builder (see the comment related with the builder)